python/tritonserver/_api/_datautils.py

```diff
@@ -176,6 +176,7 @@ def __getitem__(self, key):
     "Triton server data type",
     {
         bool: DataType.BOOL,
+        numpy.bool_: DataType.BOOL,
```

Comment:
Would be nice to start iteratively adding some unit tests for some of these, would it be an easy addition or copy of [test_request_create](https://github.com/triton-inference-server/core/blob/06b8f6e98fa90f93c02d899886564a6e88f2ae02/python/test/test_api.py#L84) by passing in a `parameters` dict with a float?